Makefile

```diff
@@ -218,3 +218,9 @@ run-demo:
 	$(MAKE) run-camera-simulator
 	@echo Running Object_detection gRPC pipeline
 	PIPELINE_PROFILE="object_detection" RENDER_MODE=1 sudo -E ./run.sh --platform core --inputsrc rtsp://127.0.0.1:8554/camera_1
+
+run-cached-demo:
+	@echo "Running camera simulator"
+	$(MAKE) run-camera-simulator
+	@echo Running Object_detection gRPC pipeline
+	PIPELINE_PROFILE="object_detection" RENDER_MODE=1 sudo -E ./run.sh --platform core --inputsrc rtsp://127.0.0.1:8554/camera_1
```

Comment:
why not use camera_0 as it is first video file to use in case user to put own media file?

Reply:
currently camera0 is for barcode use case i believe

Reply:
also, run-demo is using camera1: https://github.com/intel-retail/automated-self-checkout/pull/435/files#diff-76ed074a9305c04054cdebb9e9aad2d818052b07091de1f20cad0bbac34ffb52L220

Reply:
so we better not change it